manifests/init.pp

```diff
@@ -13,7 +13,6 @@
   $ssh_config_hash_known_hosts         = 'USE_DEFAULTS',
   $ssh_config_path                     = '/etc/ssh/ssh_config',
   $ssh_config_owner                    = 'root',
-  $ssh_config_group                    = 'root',
```

Comment:
Please don't remove parameters, this is the same as changing the API and requires a major version change.